libcloudforensics/providers/azure/forensics.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Forensics on Azure."""
+
+from typing import TYPE_CHECKING, Optional
+
+from libcloudforensics.providers.azure.internal import account
+from libcloudforensics.providers.azure.internal import common
+
+if TYPE_CHECKING:
+  from libcloudforensics.providers.azure.internal import compute
+
+
+def CreateDiskCopy(
+    subscription_id: str,
+    instance_name: Optional[str] = None,
+    disk_name: Optional[str] = None,
+    disk_type: str = 'Standard_LRS') -> 'compute.AZDisk':
```

Comment:
We're not actually importing the `compute` package since we only use `AZDisk` for type hinting. We therefore use forward reference (see [PEP 484](https://www.python.org/dev/peps/pep-0484/#forward-references)) to declare the type. Removing the quotes will break `mypy`.

Reply:
Thanks for the explanation Theo!